lib/fastly_nsq/manager.rb

```diff
@@ -20,11 +19,11 @@ class FastlyNsq::Manager
   #
   # @param logger [Logger]
   # @param pool_options [Hash] Options forwarded to {FastlyNsq::PriorityThreadPool} constructor.
-  def initialize(logger: FastlyNsq.logger, **pool_options)
+  def initialize(logger: FastlyNsq.logger, max_threads: FastlyNsq.max_processing_pool_threads, **pool_options)
```

Comment:
We should add `max_threads` to the doc string like `logger`.

Reply:
whoops, forgot to do that. good catch